db/migrate/20210803022837_create_holidays.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+class CreateHolidays < ActiveRecord::Migration[5.2]
+  def change
+    create_table :holidays do |t|
+      t.datetime :date, null: false
```

Comment:
I think that there should. be an index on this column in order to support the `.on` scope.